docs/products/kafka/kafka-connect/howto/kafka-connect-debezium-tls-pg.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Integrate Apache Kafka Connect with PostgreSQL using Debezium and mutual TLS
+---
+
+Learn how to set up Apache Kafka Connect with PostgreSQL and Debezium using mutual TLS for secure data synchronization.
+
+## Overview
+
+Integrating Apache Kafka Connect with PostgreSQL using Debezium and mutual TLS (mTLS)
+enhances data security by requiring the client and server to authenticate each other.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- Access to an Aiven for Apache Kafka service with
+  [Apache Kafka Connect](/docs/products/kafka/kafka-connect/howto/enable-connect)
+  enabled.
+- Administrative access to a PostgreSQL database with SSL enabled.
+- The following SSL certificates and keys obtained from your PostgreSQL database:
+  - SSL client certificate: The public certificate for client authentication.
+  - SSL root certificate: The certificate of the Certificate Authority (CA) that signed
+    the servers' and clients' certificates.
+  - SSL client key: The client's private key is used to encrypt the data sent to the
+    server.
+  For additional details, see
+  [Certificate requirements](/docs/platform/concepts/tls-ssl-certificates#certificate-requirements).
+
+### For CloudSQL PostgreSQL databases
+
+If you're integrating with a CloudSQL database, perform these additional steps:
+
+- **IP whitelisting**: Ensure you whitelist Aiven's IP addresses to allow connections
+  from Apache Kafka Connect to reach your CloudSQL database.
+- **Configure WAL and logical decoding**:
+  - To capture database change events, `set cloudsql.logical_decoding` to `on`.
+  - To configure the Write-Ahead Log (WAL) for logical replication, set
+    `cloudsql.enable_pglogical` to` on`.
+  - Restart the CloudSQL instance to apply changes.
```

Comment:
if these aren't already in place I assume

Reply:
@eliax1996 Could you please confirm this? Based on our previous discussions and [this](https://github.com/aiven/aiven-docs/pull/38#discussion_r1482565924), added the section for CloudSQL.

Reply:
yes @MichaelTansiniAiven its right, only if they aren't already in place